google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -241,7 +539,11 @@ Rule toPb() {
    * Delete rule class for blobs in the bucket that have been created before a certain date.
    *
    * @see <a href="https://cloud.google.com/storage/docs/lifecycle">Object Lifecycle Management</a>
+   *
+   * @deprecated Use a {@code LifecycleRule} with a {@code DeleteLifecycleAction} and use
+   * {@code LifecycleCondition.Builder.setCreatedBefore} instead
```

Comment:
Suggestion: "Use a {@code LifecycleRule} with an action {@code DeleteLifecycleAction} and a condition {@code LifecycleCondition.Builder.setCreateBefore} instead."
Request: Add a period to finish the sentence. **_Apply this to each condition deprecation message._**

You could include an additional snippet here or link back to one that you have already created.